features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplay.kt

```diff
@@ -11,6 +11,10 @@ import com.datadog.android.api.SdkCore
 import com.datadog.android.api.feature.Feature.Companion.SESSION_REPLAY_FEATURE_NAME
 import com.datadog.android.api.feature.FeatureSdkCore
 import com.datadog.android.sessionreplay.internal.SessionReplayFeature
+import com.datadog.android.sessionreplay.internal.presequite.CpuRequirementChecker
+import com.datadog.android.sessionreplay.internal.presequite.MemoryRequirementChecker
+import com.datadog.android.sessionreplay.internal.presequite.SessionReplayRequirement
+import com.datadog.android.sessionreplay.internal.presequite.SystemRequirementConfiguration
```

Comment:
presequite should be prerequisite I think

Reply:
Sorry for my English, I put it into Google translate it didn't correct me for the spell. updated now